CBLClient/Query.py

```diff
@@ -447,14 +447,12 @@ def removeChangeListener(self, query, change_listener):
         args.setMemoryPointer("changeListener", change_listener)
         return self._client.invokeMethod("query_removeChangeListener", args)
 
-    def query_selectAll(self, database):
+    def query_select_all(self, database):
```

Comment:
please sync up your branch with the latest master, this change will break some tests